tests/ext/telemetry/config.phpt

```diff
@@ -27,7 +27,11 @@ foreach (file(__DIR__ . '/config-telemetry.out') as $l) {
     if ($l) {
         $json = json_decode($l, true);
         if ($json["request_type"] == "app-started") {
-            print_r($json["payload"]["configuration"]);
+            $cfg = $json["payload"]["configuration"];
+            print_r(array_values(array_filter($cfg, function($c) {
+                return $c["origin"] == "EnvVar";
+            })));
+            var_dump(count($cfg) > 100); // all the configs, no point in asserting them all here
```

Comment:
Could be nice to assert that just the count of default values is > 100, that the count of other origins is 0

Reply:
Yes, done :-)